README.rst

```diff
@@ -179,7 +179,7 @@ You can find a quick demo of coala here:
 
 |asciicast|
 
-.. |asciicast| image:: https://asciinema.org/a/42968.png
+.. |asciicast| image:: http://i.imgur.com/rLasBoQ.png
    :target: https://asciinema.org/a/42968?autoplay=1
    :width: 100%
```

Comment:
It is the same image resized to 800 x 536. Isn't changing image size a better option than changing width?